class-debug-bar-cron.php

```diff
@@ -380,6 +381,17 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by scheldule interval
+		 *
+		 * @param array $a first element of compasion  pair
+		 * @param array $b second element of compasion  pair
+		 *
+		 * @return int 1 if element $a interval greater then element $b interval, -1  otherwise.
```

Comment:
What about when they are the same ? Different plugins might have added schedules with the *same* interval, but a different schedule name.